### PR TITLE
Add Gradle jni

### PR DESCRIPTION
**Pull request by kenvifire**
Add gradle JNI
- Generate jni headers automatically, removed manually generated files
- Move wasmedge-jni to wasmedge-java dir
- Build both jni lib and jar with `gradlew build`
- Removed duplicated Readme

### CODE REVIEW

**Comment by dannypsnl** [comment]
@kenvifire Could you resolve the conflicts? Then I can start reviewing.

**Comment by kenvifire** [comment]
@dannypsnl done!

File: bindings/java/README.md [hunk 1]
```diff
@@ -4,14 +4,11 @@
 
 ### 1. Build & install shared library
 - Follow [build.md](https://github.com/WasmEdge/WasmEdge/blob/master/docs/build.md) to build and install WasmEdge
```
Comment (dannypsnl) [inline bindings/java/README.md:6]: This is a dead link, should be https://wasmedge.org/book/en/contribute/build_from_src.html now

**Review by dannypsnl — CHANGES_REQUESTED** [review]

File: bindings/java/README.md (hunk 1 above)
Comment (kenvifire) [reply bindings/java/README.md]: Thanks for your review, fixed them.

**Review by dannypsnl — DISMISSED** [review]

**Comment by dannypsnl** [comment]
This seems good, @kenvifire please sync the master branch, then we should able to merge it. cc @hydai

**Comment by kenvifire** [comment]
> This seems good, @kenvifire please sync with the master branch, then we should able to merge it. cc @hydai

Synced.

**Review by hydai — APPROVED** [review]
LGTM

**Pull request merged**